### DISCUSSION TITLE
Project 4 question 2

### DISCUSSION THREAD

**Discussion by taa18009** [cooperrc/computational-mechanics#63] [Q&A]
I was hoping someone could help me with project 4, question 2, part c and d? I'm not sure where I went wrong here to get these errors.
<img width="1278" alt="Screen Shot 2022-06-27 at 11 26 13 PM" src="https://user-images.githubusercontent.com/106207930/176085745-564f40bd-a38b-4d80-bb2a-002c732f77b1.png">
<img width="1210" alt="Screen Shot 2022-06-27 at 11 26 22 PM" src="https://user-images.githubusercontent.com/106207930/176085728-cbcb0417-cb7e-4f9c-9630-d771d85779ae.png">
<img width="1219" alt="Screen Shot 2022-06-27 at 11 26 34 PM" src="https://user-images.githubusercontent.com/106207930/176085740-cbef040b-7c09-4618-8fac-c184b21dc9fb.png">

**Comment by cooperrc** [comment]
the first error, 

```python 
ValueError: cannot reshape size 2 into shape (11, 1)
```

comes from the lines

```python
y = np.array([1, 1]) # size (2, )
Aug = np.block(A, y.reshape(n, 1)) 
```
here n == 11, but you only have 2 elements defined

The error with `'nodes' is not defined` goes away if you run

```python
fea_arrays = np.load('./fea_arrays.npz')
K=fea_arrays['K']
nodes = fea_arrays['nodes']
```
